### ISSUE TITLE
DeepL target_lang for EN-GB and EN-US

### ISSUE THREAD

**Issue by unreadableusername** [wagtail/wagtail-localize#738]
The [DeepL API documentation](https://www.deepl.com/de/docs-api/translate-text/translate-text) specifies to use EN-GB or EN-US to archieve the desired translation, however following code in the deepl.py will chop off the -GB or -US and the deprecated unspecified route will be used.

```python
def language_code(code, is_target=False):
    # DeepL supports targeting Brazillian Portuguese but doesn't have this for other languages
    if is_target and code in ["pt-pt", "pt-br"]:
        return code
    return code.split("-")[0].upper()
```

I suggest to add en-gb and en-us to the array inside the if statement above to respect users desired locale setting.
Please tell me if this is shortsightet or more complicated then this 😅

**Comment by zerolab** [comment]
@unreadableusername I think the note on `target_lang` - "EN - English (unspecified variant for backward compatibility; please select EN-GB or EN-US instead)" is fairly clear. We should follow the advised approach. Do you have the capacity to submit a PR?

**Comment by enzedonline** [comment]
You'll get me started on a rant about EN being taken to mean EN-US when US English is a dialect of English only used in parts of North America 😂. European (and more) sites that use EN mean international English which definitely is not US English. It's a bit like using FR and getting Quebecois instead.

I've always thought there should be a configurable default locale when using language codes. Having to specify something like EN-GB in the url to get non-American English makes no sense and makes the reader it's something UK specific.

**Comment by zerolab** [comment]
Added tests and merged in a3efa8f
 ·
